packages/devtools_app/lib/src/memory/memory_events_pane.dart

```diff
@@ -127,6 +137,9 @@ class MemoryEventsPaneState extends State<MemoryEventsPane>
   /// Controller for managing memory collection.
   MemoryController _memoryController;
 
+  /// Flutter events and user custom events.
+  static const extensionEvent = 3.7;
```

Comment:
comment on what  3.7 and 2.4 mean here and why they were chosen.

Reply:
```
  /// Note: The event pane is a fixed size chart (y-axis does not scale). The
  ///       Y-axis fixed range is (visibleVmEvent to extensionEvent) e.g.,
  /// 
  ///                         ____________________
  ///         extensionEvent -|            * (3.7)
  ///                         |         * (2.4)
  ///                         |      * (1.4)
  ///         visibleVmEvent -|__ *  (0.4)
  /// 
  ///       The *s in the above chart are plotted at each y position (3.7, 2.4, 1.4, 0.4).
  ///       Their y-position is such that the symbols won't overlap.
  /// TODO(terry): Consider a better solution percent location in the Y-axis
  ///              might be a better interface.
  
  /// Flutter events and user custom events.
  static const extensionEvent = 3.7;

  /// Event to display in the event pane (User initiated GC, snapshot,
  /// automatic snapshot, etc.)
  static const visibleEvent = 2.4;

  /// Monitor events Y axis.
  static const visibleMonitorEvent = 1.4;

  /// VM's GCs are displayed in a smaller glyph and closer to the heap graph.
  static const visibleVmEvent = 0.4;
```